src/Beta/Microsoft/Graph/Model/Legalhold.php

```diff
@@ -0,0 +1,401 @@
+<?php
+/**
+* Copyright (c) Microsoft Corporation.  All Rights Reserved.  Licensed under the MIT License.  See License in the project root for license information.
+* 
+* Legalhold File
+* PHP version 7
+*
+* @category  Library
+* @package   Microsoft.Graph
+* @copyright © Microsoft Corporation. All rights reserved.
+* @license   https://opensource.org/licenses/MIT MIT License
+* @link      https://graph.microsoft.com
+*/
+namespace Beta\Microsoft\Graph\Model;
+
+/**
+* Legalhold class
+*
+* @category  Model
+* @package   Microsoft.Graph
+* @copyright © Microsoft Corporation. All rights reserved.
+* @license   https://opensource.org/licenses/MIT MIT License
+* @link      https://graph.microsoft.com
+*/
+class Legalhold extends Entity
```

Comment:
legalHoldStatus is camelCase, but legalhold isn't. I hope they finalize the naming before this goes to v1 to avoid breaking change.

Reply:
https://graph.microsoft.com/beta/compliance/ediscovery/?$whatif, I'll them know